src/store/actions/profileActions.js

```diff
@@ -174,6 +174,75 @@ export const getUserProfileData = handle => async (
   }
 };
 
-export const clearUserProfile = () => dispatch => {
+export const clearUserProfile = () => (dispatch) => {
   dispatch({ type: actions.CLEAR_USER_PROFILE_DATA_STATE });
 };
+
+export const addUserFollower = (
+  currentProfileData,
+  followers,
+  following,
+  profileData,
+  firestore,
+  dispatch
+) => {
+  try {
+    if (followers && followers.includes(currentProfileData.handle)) {
+      console.log("already followed");
```

Comment:
Can you please clean up this unnecessary console.log (if you don't need it anymore for development purposes)?

Reply:
I thought It will be still in development because after follow we still need to show all activity to the dashboard.nvm, I am removing this console from code.

Reply:
removed.